python/uv/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+The Python API of `uv` is not guaranteed to be stable and may chance at any time. It is 
+strongly discouraged to import from our `uv` Python module, but you can do so at your 
+own risk.
+"""
+from uv.__main__ import detect_virtualenv, find_uv_bin

```

Comment:
Let's not import then from `__main__`, let's do it the other way around (define them here and import them there).

I think `detect_virtualenv` should remain private as well.